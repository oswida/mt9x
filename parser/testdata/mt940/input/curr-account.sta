:20:0806120035300121
:25:44123456
:28C:112/1
:60F:C080611NOK116,98
:61:0806110611C2500000,NTRF22233300/6000//A019910450123456
NOLI
:61:0806110611D2500000,NTRFTW100012//PU00459450123456
60000-IT-A06
:62F:C080611NOK116,98
:64:C080611NOK116,98