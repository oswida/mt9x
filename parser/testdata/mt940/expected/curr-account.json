{
 "tag20": "0806120035300121",
 "tag25": {
  "account": "44123456"
 },
 "tag28": {
  "stmt_number": "112",
  "seq_number": "1"
 },
 "tag60": {
  "dc_mark": "C",
  "date": "2008-06-11T00:00:00Z",
  "currency": "NOK",
  "amount": "116.98"
 },
 "statements": [
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "2500000",
    "trx_ident": "NTRF",
    "owner_ref": "22233300/6000",
    "institution_ref": "A019910450123456",
    "details": "NOLI"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "2500000",
    "trx_ident": "NTRF",
    "owner_ref": "TW100012",
    "institution_ref": "PU00459450123456",
    "details": "60000-IT-A06"
   }
  }
 ],
 "tag62": {
  "dc_mark": "C",
  "date": "2008-06-11T00:00:00Z",
  "currency": "NOK",
  "amount": "116.98"
 },
 "tag64": {
  "dc_mark": "C",
  "date": "2008-06-11T00:00:00Z",
  "currency": "NOK",
  "amount": "116.98"
 }
}